test/python/apps/exchange_navigation_helper.py

```diff
@@ -0,0 +1,62 @@
+from pathlib import Path
+
+from ragger.navigator import Navigator, NavInsID
+from ragger.backend import BackendInterface
+
+ROOT_SCREENSHOT_PATH = Path(__file__).parent.parent.resolve()
+
+class ExchangeNavigationHelper:
```

Comment:
The `Navigator` is also an helper firstly (as most if not all abstraction code in general). Naming this class `ExchangeNavigator` feels enough for me. I'm no big fan of `LongNameWithoutMuchValueAddedInIt`